app/utils/ErrorHandler.scala

```diff
@@ -1,44 +0,0 @@
-package utils
-
-import javax.inject.Inject
-
-import com.mohiva.play.silhouette.api.SecuredErrorHandler
-
-import play.api.http.DefaultHttpErrorHandler
-import play.api.i18n.{I18nSupport,Messages,MessagesApi}
-import play.api.mvc.Results._
-import play.api.mvc.{Result,RequestHeader}
-import play.api.routing.Router
-import play.api.{OptionalSourceMapper,Configuration}
-
-import services.DispenserService
-
-import scala.concurrent.Future
-
-import controllers.routes
-
-class ErrorHandler @Inject() (
-  val messagesApi: MessagesApi,
-  env: play.api.Environment,
-  config: Configuration,
-  sourceMapper: OptionalSourceMapper,
-  dispenserService: DispenserService,
-  router: javax.inject.Provider[Router])
-  extends DefaultHttpErrorHandler(env, config, sourceMapper, router)
-  with SecuredErrorHandler with I18nSupport {
-
-  override def onNotAuthenticated(request: RequestHeader, messages: Messages): Option[Future[Result]] =
```

Comment:
Currently, silhouette handles the `SecuredAction` instances in the controller package (also in its `webapp` subpackage). I'm not sure, if we handle errors on another way for current controller implementations, but this would be the default handler. Thus, if we want to delete it, we should check for the whole controller package (including all subpackages) that there exists an alternative handler.